--- a/misc.py	2026-07-20
+++ b/misc.py	2026-07-20
@@ -90,9 +90,9 @@
     'logs': 'C:\\Certbot\\log',
 }
 LINUX_DEFAULT_FOLDERS = {
-    'config': '/etc/letsencrypt',
-    'work': '/var/lib/letsencrypt',
-    'logs': '/var/log/letsencrypt',
+    'config': '/opt/zextras/common/certbot/etc/letsencrypt',
+    'work': '/var/lib/carbonio-letsencrypt',
+    'logs': '/var/log/carbonio/letsencrypt',
 }
 
 def get_default_folder(folder_type: str) -> str:
